src/main/java/org/apache/bcel/classfile/ClassParser.java

```diff
@@ -59,7 +59,7 @@ public final class ClassParser {
     /**
      * Parses class from the given stream.
      *
-     * @param inputStream Input stream
+     * @param inputStream Input stream; the client should close it
```

Comment:
-1: No need to say this IMO, it should be a basic best practice that the allocation site is the closing site. Only the opposite should be documented.